File: problem-set-1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Do-while loop for user's input.
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    //While loop with two for loops nested within for printing a pyramid.
    int x = 1;
    while (x <= h)
    {
        for (int a = h - x; a > 0; a--)
        {
            printf(" ");
        }
        for (int b = 1; b <= x; b++)
        {
            printf("#");
        }
        printf("\n");
        x++;
    }
}

File: problem-set-6/mario/less/mario.py
import cs50

# Improvised do-while loop for users input
h = cs50.get_int("Height: ")

while h < 1 or h > 8:
    h = cs50.get_int("Height: ")


# While loop for printing a pyramid.
h2 = h

while h2 != 0:
    i = h2 - 1
    print(" " * i + "#" * (h - i))
    h2 -= 1
